src/resource-graph/azext_resourcegraph/commands.py

```diff
@@ -4,16 +4,22 @@
 # --------------------------------------------------------------------------------------------
 
 from azure.cli.core.commands import CliCommandType
-from ._client_factory import cf_resource_graph
+from ._client_factory import cf_resource_graph, cf_resource_graph_graph_query
 from ._validators import validate_query_args
 
 
 def load_command_table(self, _):
 
-    graph_sdk = CliCommandType(
-        operations_tmpl='azext_resourcegraph.custom#{}',
-        client_factory=cf_resource_graph
+    graph_shared_query_sdk = CliCommandType(
+        operations_tmpl='azext_resourcegraph.vendored_sdks.resourcegraph.operations#GraphQueryOperations.{}',
+        client_factory=cf_resource_graph_graph_query
     )
 
-    with self.command_group('graph', command_type=graph_sdk, client_factory=cf_resource_graph) as g:
+    with self.command_group('graph', client_factory=cf_resource_graph) as g:
         g.custom_command('query', 'execute_query', validator=validate_query_args)
+
+    with self.command_group('graph shared-query', graph_shared_query_sdk) as g:
```

Comment:
Have you synced with Arun with the commands as he suggested to use "az graph query share -n -g -q" to be consistent with Azure portal?

Reply:
`az graph query` has already be a command for a while, we cannot have a command group called `az graph query`.

Reply:
It's better comment this in speclet and @arun to let him know this change.

---
In reply to: [396180119](https://github.com/Azure/azure-cli-extensions/pull/1400#discussion_r396180119) [](ancestors = 396180119)

Reply:
Done